### DISCUSSION TITLE
Client connection timestamp

### DISCUSSION THREAD

**Discussion by Gianluca-Casagrande-Stiga** [moscajs/aedes#919] [Q&A] [answered]
I would like to have the snapshot of all connected client Ids, with their related connection duration.
Is there a way to get the last connection timestamp from a client instance? If not, could it be an interesting information to add?

**Comment by gnought** [comment, marked as answer]
It's quite an interesting idea. Imo storing the session start timestamp per client as a base is good enough, that reserve rooms for developers to adapt in their use cases.

**Reply by Gianluca-Casagrande-Stiga** [reply]
I resolved my issue adding that information to the client, inside "authenticate" function.
